config_jira_notifications.go

```diff
@@ -0,0 +1,46 @@
+package evergreen
+
+import (
+	"text/template"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+)
+
+type JIRANotificationsConfig struct {
+	CustomFields map[string]JIRAProjectFields `bson:"custom_fields"`
```

Comment:
I think we wanted to avoid user-specified keys in the db. Double check with Sam, but I think this needs to be a slice of structs

Reply:
agreed.